internal/distributed/proxy/service.go

```diff
@@ -162,6 +162,14 @@ func (s *Server) startExternalGrpc(grpcPort int, errChan chan error) {
 	}
 	log.Debug("Proxy server already listen on tcp", zap.Int("port", grpcPort))
 
+	multiLimiter := proxy.NewMultiRateLimiter()
```

Comment:
Create RateLimiter in internal/proxy. 
and Add GetRateLimiter to  proxy interface